hastexo/static/html/main.html

```diff
@@ -9,6 +9,7 @@
     <select class="port"></select>
     <input type="button" value="Reset" class="reset" />
     <input type="button" value="Check progress" class="check" />
+    <input type="button" value="Open in new window" class="launch"/>
```

Comment:
Another nit: I'd love to keep the "Check Progress" button the rightmost. So I think we'll want the "Open in new window" button perhaps left-aligned below the terminal?

Reply:
Done.